style.css

```
@@ -0,0 +1,778 @@
+    animation-duration: 5s;
+    animation-iteration-count: infinite;
+
+}
+.stars_3::after{
+    content: "";
+    width: 3px; 
+    height: 3px;
+    top: -190px;
+    left: 330px;
+    background-color: white;
+    position: absolute;
+    border-radius: 50px;
+    animation-name: stars;
+    animation-duration: 8s;
+    animation-iteration-count: infinite;
+}
+
+.stars_4{
+    width: 8px;
+    height: 8px;
+    top: 450px;
+    left: 450px;
+    position: absolute;
+    border-radius: 10px;
+    background-color: #fff;
+    animation-name: stars;
+    animation-duration: 4s;
+    animation-iteration-count: infinite;
+}
+.stars_4::before{
+    content: "";
+    width: 8px; 
+    height: 8px;
+    top: 0px;
+    left: -370px;
+    background-color: white;
+    position: absolute;
+    border-radius: 50px;
+    animation-name: stars;
+    animation-duration: 5s;
+    animation-iteration-count: infinite;
+}
+.stars_4::after{
+    content: "";
+    width: 5px; 
+    height: 5px;
+    top: -400px;
+    left: -390px;
+    background-color: white;
+    position: absolute;
+    border-radius: 50px;
+    animation-name: stars;
+    animation-duration: 5s;
+    animation-iteration-count: infinite;
+}
+/* PLANET_FILL */
+.planet{
+    width: 250px;
+    height: 250px;
+    border-radius: 200px;
+    top: 125px;
+    left: 135px;
+    position: absolute;
+    overflow: hidden;
+    z-index: 1;
+}
+.planet_fill_1{
+    width: 300px;
+    height: 300px;
+    background-color: #1165a6;
+    border-radius: 200px;
+    top: -80px;
+    left: -50px;
+    position: absolute;
+}
+.planet_fill_2{
+    width: 250px;
+    height: 250px;
+    background-color: #07355a;
+    border-radius: 200px;
+}
+.planet_fill_3{
+    width: 250px;
+    height: 250px;
+    border-radius: 200px;
+    background-color: #1476bc;
+    top: -70px;
+    left: -30px;
+    position: absolute;
+}
+/* EARTH_ISLAND_1*/
+.earth_1{
+    top: 0px;
+    left:30px;
+    position: absolute;
+    width: 60px;
+    height: 130px;
+    background-color: rgb(45, 177, 45);
+    z-index: 1000;
+}
+.earth_1::after{ 
+    content: "";
+    position: absolute;
+    width: 110px;
+    height: 40px;
+    background-color: rgb(45, 177, 45);
+    top: 0px;
+    left: 0px;
+    border-radius: 50px 0;
+}
+.earth_1::before{
+    content: "";
+    position: absolute;
+    width: 150px;
+    height: 35px;
+    background-color: rgb(45, 177, 45);
+    top: 65px;
+    left: -30px;
+    border-radius: 30px;
+}
+.stick_horison_1{
+    position: absolute;
+    width: 90px;
+    height: 35px;
+    background-color: rgb(45, 177, 45);
+    top: 125px;
+    left: -14px;
+    border-radius: 30px;
+}
+.stick_right_1{ 
+    position: absolute;
+    width: 30px;
+    height: 25px;
+    background-color: #1476bc;
+    top: 40px;
+    left: 50px;
+    border-radius: 40%;
+    z-index: 10001;
+}
+.stick_right_1::after{
+    position: absolute;
+    content: "";
+    width: 30px;
+    height: 25px;
+    background-color: #1476bc;
```

Используйте [переменные](https://developer.mozilla.org/ru/docs/Web/CSS/Using_CSS_custom_properties) для цветов и прочих константных значений.
Для простоты можете в самом начале файла создать рутовый селектор и объявлять переменные в нём
```
:root {
  --some-color-i-like: #1476bc;
}
```
```suggestion
    background-color: var(--some-color-i-like);
```

---

Если захотите что-то изменить - достаточно будет изменить переменную.